src/common/low_precision_transformations/src/pad.cpp

```diff
@@ -179,10 +181,6 @@ bool PadTransformation::canBeTransformed(const TransformationContext& context, s
         return false;
     }
 
-    if (hasNegativeIndexes(pad)) {
-        return false;
-    }
-
     const auto dequantization = NetworkHelper::getDequantization(op, defaultPrecisions);
     if (dequantization.empty()) {
         return false;
```

Comment:
Below we have checks for Constant and Reflect pad modes: they are actual only in case of positive or mixed indexes. If all indexes are negative, we can return true before these checks